File: aoj/itp1/js/2-c-sorting-three-numbers.js
const rl = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("", (args) => {
  let tmp;
  let [a, b, c] = args.split(" ").map(Number);
  tmp = a;
  if (a > b) {
    a = b;
    b = tmp;
  }
  if (a > c) {
    tmp = a;
    a = c;
    c = tmp;
  }
  if (b > c) {
    tmp = b;
    b = c;
    c = tmp;
  }
  console.log(`${a} ${b} ${c}`);

  rl.close();
})

File: py/aoj/itp1/2-c-sorting-three-numbers.py

# 3つ受け取る
a, b, c = map(int, input().split(" "))

# aとbを比較
if a > b:
    a, b = b, a

# aとcを比較
if a > c:
    a, c = c, a
    
# bとcを比較
if b > c:
    b, c = c, b

print(f"{a} {b} {c}")